### ISSUE TITLE
spaces in entityUri not encoded for underlying SPARQL query

### ISSUE THREAD

**Issue by mmisw** [mmisw/mmiorr#323]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on July 15, 2014 10:32:29_

What steps will reproduce the problem? 

1.  make HTTP request for a term having character encoded, eg: 
    http://mmisw.org/ont/mmi/someVocab/some%20Term

What is the expected output/behavior? 
Normal processing 

What do you see instead? 
```
MALFORMED QUERY: Found \<. Was expecting one of: BIND, BLANK_NODE_LABEL,
DECIMAL, DOUBLE,
FALSE, FILTER, GEO, GRAPH, INTEGER, MINUS, NIL-SYMBOL, OPTIONAL,
Q_IRI_REF, QNAME, QNAME_NS, SELECT, SERVICE, STRING_LITERAL1,
STRING_LITERAL2, STRING_LITERAL_LONG1, STRING_LITERAL_LONG2, TEXTINDEX,
TRUE, VALUES, VARNAME or punctuation '(', '+', '-', '[', '[]', '{', '}'.
```
Seems like there's a missing encoding of the entityUri for the underlying SPARQL query.

What version of the product are you using?
MMI Ontology and Term URI Resolver  2.1.1 (201407151150)

_Original issue: http://code.google.com/p/mmisw/issues/detail?id=323_

**Comment by mmisw** [comment]
_From [caru...@gmail.com](https://code.google.com/u/113886747689301365533/) on July 15, 2014 10:43:11_

The concrete problem is with spaces, eg: http://mmisw.org/ont/mmitest/udunits2-accepted/ång ström
or, equivalently: http://mmisw.org/ont/mmitest/udunits2-accepted/ång%20ström (note that request with any other special characters eg http://mmisw.org/ont/mmitest/udunits2-accepted/ångström is just fine).

**Summary:** spaces in entityUri not encoded for underlying SPARQL query  \<span class="oldvalue"> (was: entityUri not encoded for underlying SPARQL query) \</span>

**Comment by carueda** [comment]
This is fixed, see (duplicate) #366

I completely forgot about this older entry and didn't do my homework of searching first!
